### DISCUSSION TITLE
deleted was not looking for responses by random people, rather lichess

### DISCUSSION THREAD

**Discussion by igroti** [lichess-org/mobile#12] [General]
deleted
was not looking for responses by random people, rather lichess

**Comment by prolificcoder** [comment]
> using a flutter app for example YouTube

Do you have a source for this?

**Reply by igroti** [reply]
deleted
was not looking for responses by random people, rather lichess

**Reply by egecavusoglu** [reply]
YouTube does not use Flutter, uses native iOS with ObjC. Nevertheless, betting on Flutter is a big design decision by the lichess team. I'm sure given budget and dev team constraints they had to go with a cross platform decision, but nonetheless it is a bold decision. Hope it scales well. Wondering what kind of a stack the current mobile uses, I was pretty happy with it.

**Reply by jonbgamble** [reply]
The previous mobile was typescript/javascript and styl/css with [capacitor](https://capacitorjs.com/docs/) providing some native features.

**Reply by egecavusoglu** [reply]
I see, then why not use a lot of that ts/js logic (realtime connections and etc.) through React Native?

**Comment by egecavusoglu** [comment]
Lol @DaniSpringer the beauty of internet is that it's run by random people
